src/js/pre-need/config/form.js

```diff
@@ -454,21 +455,26 @@ const formConfig = {
             application: {
               claimant: {
                 desiredCemetery: autosuggest.uiSchema(
-                  'Which VA National Cemetery would you prefer to be buried in?',
+                  'Which VA national cemetery would you prefer to be buried in?',
                   getCemeteries
                 ),
                 'view:desiredCemeteryNote': {
-                  'ui:description': (
-                    <div className="usa-alert usa-alert-info no-background-image">
-                      <strong>Please note:</strong> This doesn't guarantee you'll be buried in your preferred cemetery. We'll try to fulfill your wishes, but will assign a gravesite in a cemetery with available space at the time of need.
-                    </div>
-                  )
+                  'ui:description': desiredCemeteryNote
                 }
               },
               hasCurrentlyBuried: {
-                'ui:title': 'Is there anyone currently buried in a VA National Cemetery under your eligibility?',
                 'ui:widget': 'radio',
                 'ui:options': {
+                  updateSchema: (formData, schema) => {
+                    if (isVeteran(formData)) {
+                    /* eslint-disable no-param-reassign */ 
+                      schema.title = 'Is there anyone currently buried in a VA national cemetery under your eligibility?';
```

Comment:
You don't have to return the whole schema here, just, the updated property, so you can do this without mutating the schema:

```js
let title;
if (isVeteran(formData)) {
  title = 'Is there anyone currently buried in a VA national cemetery under your eligibility?';
} else {
  title = 'Is there anyone currently buried in a VA national cemetery under your sponsor’s eligibility?';
}

return { title };
```